### DISCUSSION TITLE
[NFTs] Sold out on LE

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#1865] [General]
I put a lot of effort into getting NFTs before the land expansion. I gave up some things to achieve these goals. The NFTs as decoration, buffs and flags that were in V2 will no longer be available right? The only ones that will be available in the LE are the mutant chicken so the rooster doesn't lose its function right?

Thank you.

**Comment by adamhannigan** [comment]
@future2back Every asset you collected in beta you can use in Sunflower Isles.

Every rare item & skill that was launched in Beta, is no longer available for new players. Only OG players could get these.

The Rooster is one of these items no longer available for new players to craft. 

Mutant Chickens are still available though and if you have a Rooster, the boost still applies

**Reply by future2back** [reply]
It was worth my effort to acquire flag, rooster and my nfts.


Thank you for replying @adamhannigan.

**Reply by future2back** [reply]
hey @adamhannigan could you please update this information for legacy items on Open Sea?
<s> You can craft this item at the Goblin Tailor in the game.</s>  Sold out!
<s> You can craft this item at the Goblin Blacksmith in the game.</s> Sold out!

This information is very important for new and old players.  Owners must know what they have and buyers what they are buying.


I also believe that you could add a border and a mark on legend items to differentiate.
Just to illustrate:

![image](https://user-images.githubusercontent.com/74167741/208002626-ebd71597-07f6-499f-90a7-60917867fa8c.png)


These adjustments can help identify the true value of these items. What do you think about this? I will be attentive to your comments.




Thank you.
